packages/service/tests/acceptance/conftest.py

```diff
@@ -9,3 +13,10 @@
 def pin_map_directory(test_assets_directory: pathlib.Path) -> pathlib.Path:
     """Test fixture that returns the pin map directory."""
     return test_assets_directory / "acceptance" / "session_management"
+
+
+@pytest.fixture
+def filter_wrong_configurations_message_type_warnings() -> Generator:
```

Comment:
There's already a mark for this: https://docs.pytest.org/en/6.2.x/warnings.html#pytest-mark-filterwarnings

Reply:
Yes, but I didn't want to have this repeated everywhere:
```py
@pytest.mark.filterwarnings(
    "ignore:.*Configurations.*:ni_measurement_plugin_sdk_service.measurement.WrongMessageTypeWarning"
)
```
(You have to use the fully-qualified name when it's not a built-in warning, and the line then becomes too long to fit onto one line.)

If you disagree, I can switch it.

Reply:
Ok, that seems like a good reason.